jenkins/custom-ci/centos-7.9.2009/csm-agent.groovy

```diff
@@ -57,8 +57,9 @@ pipeline {
 
                 sh label: 'Configure yum repository for cortx-py-utils', script: '''
                     CORTX_UTILS_REPO_OWNER=$(echo $CORTX_UTILS_URL | cut -d "/" -f4)
-
-                    yum-config-manager --nogpgcheck --add-repo=http://cortx-storage.colo.seagate.com/releases/cortx/github/kubernetes/centos-7.9.2009/last_successful_prod/cortx_iso/
+                    
+                    yum-config-manager --add-repo=http://cortx-storage.colo.seagate.com/releases/cortx/github/integration-custom-ci/$os_version/$release_tag/cortx_iso/
+                    yum-config-manager --save --setopt=cortx-storage*.gpgcheck=1 cortx-storage* && yum-config-manager --save --setopt=cortx-storage*.gpgcheck=0 cortx-storage*
```

Comment:
Why are we enabling and disabling gpgcheck? we just need to disable it right?

Reply:
yes right. But when I have taken a reference from other script https://github.com/Seagate/cortx-re/blob/main/jenkins/custom-ci/centos-7.9.2009/hare.groovy. here we did like this

Reply:
@shailesh-vaidya, Any idea on this?

Reply:
This is due to the way `setopt` work. You need to enable it first and then disable it.